skabase/SKALogger/setup.py

```diff
@@ -5,6 +5,8 @@
 #
 #
 #
+# Distributed under the terms of the none license.
```

Comment:
These lines should be removed.

Reply:
This is an outdated change. It's not there in recent code.